home-work-app/src/app/redux/actions/addMovie.ts

```diff
@@ -0,0 +1,9 @@
+import { IApiResponse } from '../../interfaces/IApiResponse';
+import { AddAction, IMovieActions } from '../../interfaces/IMovieActions';
+
+export const deleteMovie: IMovieActions.Add = (formData: IApiResponse.IMovie): AddAction => {
+    return {
+        type: 'ADD_ITEM',
```

Comment:
It;s a good practice (avoid "magic numbers") to move action types into a separate variables, since it will be used in `action` and in `reducer`
const ADD_ITEM = 'ADD_ITEM';